src/Diagnostics.RuntimeHost/Controllers/Site/SitesController.cs

```diff
@@ -76,7 +80,50 @@ public SitesController(IStampService stampService, ISiteService siteService, ICo
             
             App app = await GetAppResource(subscriptionId, resourceGroupName, siteName, postBody, startTimeUtc, endTimeUtc);
             OperationContext<App> context = PrepareContext<App>(app, startTimeUtc, endTimeUtc);
-            return await base.GetDetector<App>(detectorId, context);
+            return await base.GetDetectorResponse<App>(detectorId, context);
+        }
+
+        [HttpPost(UriElements.Insights)]
+        public async Task<IActionResult> GetInsights(string subscriptionId, string resourceGroupName, string siteName, [FromBody] DiagnosticSiteData postBody, string supportTopicId, string startTime = null, string endTime = null, string timeGrain = null)
+        {
+            if (postBody == null)
+            {
+                return BadRequest("Post Body missing.");
+            }
+
+            if (siteName == "steve")
```

Comment:
Forgot to remove this?  I guess this was for testing purpose

Reply:
I mentioned in the teams chat that I had this just for testing and I will remove :)